build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1065,14 +1065,15 @@ message OutputFile {
 // [Directory][build.bazel.remote.execution.v2.Directory] protos in a
 // single directory Merkle tree, compressed into one message.
 message Tree {
-  // The root directory in the tree.
-  Directory root = 1;
+  reserved 1, 2; // Used for removed fields in an earlier version of the API.
+
+  // The digest of the root directory in the tree.
+  Digest rootDigest = 3;
 
   // All the child directories: the directories referred to by the root and,
```

Comment:
Would the new map only contain the child directories as per the comment or also the root `Directory`? The former seems odd to me as it would require an extra round trip for the root directory but not for its child directories.

Reply:
It contains the root directory, which is alluded to (and was confusing) in the children comment previously, that it should contain everything under a root.